cmd/agent/subcommands/run/command_windows.go

```diff
@@ -95,6 +96,7 @@ func StartAgentWithDefaults(ctxChan <-chan context.Context) (<-chan error, error
 			invChecks inventorychecks.Component,
 			_ netflowServer.Component,
 			agentAPI internalAPI.Component,
+			pkgSigning packagesigning.Component,
```

Comment:
```suggestion
		_ packagesigning.Component,
```
As the test is not supposed to run on windows I'm not sure if I can write it this way